android-patches/patches/OfficeRNHost/ReactAndroid/src/main/java/com/facebook/react/bridge/ReactBridge.java

```diff
@@ -0,0 +1,25 @@
+--- a/ReactAndroid/src/main/java/com/facebook/react/bridge/ReactBridge.java
++++ b/ReactAndroid/src/main/java/com/facebook/react/bridge/ReactBridge.java
+@@ -31,6 +31,22 @@ public class ReactBridge {
+     Systrace.beginSection(
+         TRACE_TAG_REACT_JAVA_BRIDGE, "ReactBridge.staticInit::load:reactnativejni");
+     ReactMarker.logMarker(ReactMarkerConstants.LOAD_REACT_NATIVE_SO_FILE_START);
++
++    // JS Engine is configurable.. And we exepct only one packaged
++    // Hence ignore failure
++    try {
++	SoLoader.loadLibrary("hermes");
++    }catch (UnsatisfiedLinkError jscE){}
++
++    try {
++	SoLoader.loadLibrary("v8jsi");
++    }catch (UnsatisfiedLinkError jscE){}
++
++    SoLoader.loadLibrary("glog_init");
++    SoLoader.loadLibrary("fb");
++    SoLoader.loadLibrary("fbjni");
++    SoLoader.loadLibrary("yoga");
++    SoLoader.loadLibrary("jsinspector");
+     SoLoader.loadLibrary("reactnativejni");
```

Comment:
@mganandraj I think we should load the following .so's as well here
gloag,folly_json, reactperfloggerjni

As it was not there in original patch I didn't add it. 
Any specific reason for leaving these out?

Reply:
Added the libraries now in second iteration